Sources/AppStoreConnectCLI/Services/Operations/ReadBuildOperation.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ReadBuildOperation.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Nafisa Rahman on 1/5/20.
```

Comment:
Let's make the headers the same as the other files. This should happen automatically with the workspace template. Not sure what happened here